src/robusta/integrations/mattermost/client.py

```diff
@@ -39,6 +40,24 @@ def get_token_owner_id(self) -> Optional[str]:
         response_data = response.json()
         return response_data.get("user_id")
 
+    def is_admin_bot(self):
+        endpoint = "/users/me"
+        url = self._get_full_mattermost_url(endpoint)
+        response = self._send_mattermost_request(url, HttpMethod.GET)
+        if not check_response_succeed(response):
+            logging.error("Could not connect to Mattermost with bot account")
```

Comment:
Should we return here?
Will we have data in the response in case of error?

Reply:
Good point, do you think we should also check if we return `None` and early-exit the constructor?

Reply:
@arikalon1 Hoping to get this into the upcoming release 😎

Reply:
I think we should just assume we're not admin.
If it can work in a non-admin mode, no reason not to.